mmvb_backend/benchmarking_sessions/api/views.py

```diff
@@ -70,10 +76,15 @@ def benchmark_status(self, request, *args, **kwargs):
 
         strip_values_from_responses(result.info["responses"])
 
+        ais = benchmarking_session.ai_implementations.all()
+        ai_ids = [str(ai.id) for ai in ais]
+        statistics = result.info["statistics"]
+        statistics["table"] = get_stats_table(result.info["responses"], ai_ids)
+
         return Response(
             {
-                "status": BenchmarkingSession.Status.RUNNING,
-                "report": result.info,
+                "status": benchmarking_session.status,
```

Comment:
Why is this now .status and not .RUNNING?
Just for my understanding

Reply:
BenchmarkingSession.Status.RUNNING is a constant and benchmarking_session.status is the actual status in the object, before it was assumed that at this point in the code "running" is the only possible status left